packages/shared/src/utils/accountUtils.ts

```
@@ -1,5 +1,5 @@
 /* eslint-disable spellcheck/spell-checker */
-import { isNil } from 'lodash';
+import { isNaN, isNil, isNumber } from 'lodash';
```

Biome (1.9.4)

[error] Do not shadow the global "isNaN" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)